specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -964,6 +964,10 @@
             "type": "object"
           }
         },
+        "metadata": {
+          "description": "The thread metadata",
+          "type": "object"
```

Comment:
Are the values meant to be strings? You would need to add the "additionalProperties" property and include the appropriate type, e.g. https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json#L580-L582